prime-router/src/main/kotlin/cli/tests/LoadTestSimulator.kt

```diff
@@ -274,8 +287,8 @@ class Simulator : CoolTest() {
             true
         )
         val spike = Simulation( // Simulate a sudden spike in sends all at once
-            "Spike: 50 Threads each submitting once.",
-            50,
+            "Spike: 100 Threads each submitting once.",
+            100,
             1,
```

Comment:
This is an example of "amping up" the tests, so that are load tests always stress the system more than our actual production load.   We have a customer who hits us with many single submissions simultaneously., every 15 minutes. This 'spike' test is meant to simulate that ++.